products/va-mobile-app/product/significant decisions.md

```diff
@@ -510,9 +510,48 @@ The other option was to conduct load testing./
 ### When, or under what conditions, would you recommend revisiting this design decision?
 As our user base grows, we will certainly revisit the decision to not do load testing. Specifically, if usage grows by an order of magnitude more than expected, we will begin preparations to do load testing.
 
+
+## Decision 12: App Store Testing and Distribution Groups
+Once the apps are in the Google and Apple stores under the VA banner, we will need a structured way to distribute the app and specific app builds to different, disparate groups associated with the applications.
+
+### What did you decide on?
+In order to allow developers to have QA builds that may not have full functionality, but also ensure that stakeholders can see finished product, we will have multiple distribution lanes in each of the stores. The lanes are as follows:
+
+#### **Internal Team**
+The internal teams on Apple and Google will not be used unless the leads need access to every build version. This will alleviate the need to make a specific build for a group and then immediately make a new build for the internal team to use. 
+
+#### **Development Team**
+This group consists of all the engineers, UX, product and QA people on the active development team. This lane will receive scheduled builds two or three times a week, or more as needed. These builds will be automated from the tip of the development branch and may or may not include incomplete/non-functioning features. Users that are not on the immediate development team can be added to the VA Stakeholders group (below) to avoid the frequent updates this group will receive. 
+
+#### **VA Stakeholders**
+This group will have any of the VA or Ad Hoc personnel that would like to see the demo builds on their devices. This group will receive releases on the morning of demo days that are from a proscribed commit on develop. This will allow the stakeholders to see demonstrated capabilities, while minimizing the updates necessary to see the builds. It also reduces the likelihood that a non-functioning/not yet complete feature gets shared and creates confusion. 
+
+#### **VA Production Testers**
+This group will consist of VA Employees that are Veterans and who are helping through some form of long-lasting User Acceptance Testing (UAT). This group will receive updates as needed or as requested from the product team. 
+
+#### **Ad Hoc Production Testers**
+This group will consist of Ad Hoc Employees that are Veterans and who are helping through some form of long-lasting User Acceptance Testing (UAT). This group will receive updates as needed or as requested from the product team. These will be the same builds as the *VA Production Testers* above.
```

Comment:
Why have a separate lane for Ad Hoc employees, when they are really just VA Production Testers?